### PR TITLE
fix api/status cointype is empty when binded without deposit

### PR DESCRIPTION
**Pull request by zhiyuan2007**
fixed coin_type is empty only when bind without deposit 
```
curl "http://127.0.0.1:7071/api/status?skyaddr=2GLCDr78aHFb6BLJaoLBc9thGrZxDD9iTgS"
{
    "statuses": [
        {
            "seq": 0,
            "updated_at": 1514684790,
            "status": "waiting_deposit",
            "coin_type": ""
        }
 ]
}
```

### CODE REVIEW

File: src/exchange/store.go [hunk 1]
```diff
@@ -153,7 +159,7 @@ func (s *Store) BindAddress(skyAddr, depositAddr, coinType string) error {
 			}
 		}
 
-		addrs = append(addrs, depositAddr)
+		addrs = append(addrs, DepositElement{DepositAddr: depositAddr, CoinType: coinType})
```
Comment (gz-c) [inline src/exchange/store.go]: This breaks format of existing db, any other way?

File: src/exchange/store.go [hunk 2]
```diff
@@ -349,12 +355,13 @@ func (s *Store) GetDepositInfoOfSkyAddress(skyAddr string) ([]DepositInfo, error
 
 	if err := s.db.View(func(tx *bolt.Tx) error {
 		// TODO: DB queries in a loop, may need restructuring for performance
-		depositAddrs, err := s.getSkyBindBtcAddressesTx(tx, skyAddr)
+		depositElements, err := s.getSkyBindBtcAddressesTx(tx, skyAddr)
```
Comment (gz-c) [inline src/exchange/store.go]: The name is `getSkyBindBtcAddressesTx`, are they always btc addresses?

**Review by gz-c — CHANGES_REQUESTED** [review]

**Comment by gz-c** [comment]
Are bound addresses saved in a new bucket per coin type? Then can add cointype after querying values from the db

File: src/exchange/store.go (hunk 1 above)
Comment (zhiyuan2007) [reply src/exchange/store.go]: i can find address type by search used_xxx_address buckets, that is ok?

Comment (zhiyuan2007) [reply src/exchange/store.go]: or use a new bucket to store relationship between address and type

Comment (gz-c) [reply src/exchange/store.go]: Store bind addresses in a different bucket per coin, seems best

File: src/exchange/store.go [hunk 3]
```diff
@@ -368,11 +390,16 @@ func (s *Store) GetDepositInfoOfSkyAddress(skyAddr string) ([]DepositInfo, error
 			// has not sent a deposit to the exchange, so the status is
 			// StatusWaitDeposit.
 			if len(txns) == 0 {
+				coinType, err := s.getDepositAddressCoinType(depositAddr)
```
Comment (gz-c) [inline src/exchange/store.go:393]: if coinType == "", log warning

**Review by gz-c — CHANGES_REQUESTED** [review]

File: cmd/transmit/transmit.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
+package main
```
Comment (gz-c) [inline cmd/transmit/transmit.go:1]: rename transmit to dbmigrate

**Review by gz-c — CHANGES_REQUESTED** [review]

**Comment by gz-c** [comment]
need small update

**Comment by gz-c** [comment]
#190

**Pull request merged**